src/ASM/ASMs3DLag.h

```diff
@@ -207,7 +207,6 @@ class ASMs3DLag : public ASMs3D
   int    p2; //!< Polynomial order in second parameter direction
   int    p3; //!< Polynomial order in third parameter direction
 
-private:
   const std::vector<Vec3>& coord; //!< Nodal coordinates
```

Comment:
Using `Vec3 getCoord(size_t) const` method instead? Or you need to tamper with the coordinates in a sub-class

Reply:
discuss elsewhere.